test/SdkUnitTestsWithCallback.js

```diff
@@ -884,4 +885,314 @@ describe('SDK Unit Tests With Callbacks:', function (done) {
+
+    envelopesApi.listRecipients(accountId, envelopeId, function(error, recipients, _response){
+      if (error) {
+        return done(error);
+      }
+
+      assert.notStrictEqual(recipients, undefined);
+      done();
+    })
+  })
+
+  it('Create sender view', function(done) {
+    var fileBytes = null;
+    try {
+      var fs = require('fs');
+      // read file from a local directory
+      fileBytes = fs.readFileSync(path.resolve(__dirname, SignTest1File));
+    } catch (ex) {
+      // handle error
+      console.log('Exception: ' + ex);
+    }
+
+    // create an envelope to be signed
+    var envDef = new docusign.EnvelopeDefinition();
+    envDef.emailSubject = 'Please Sign my Node SDK Envelope';
+    envDef.emailBlurb = 'Hello, Please sign my Node SDK Envelope.';
+
+    // add a document to the envelope
+    var doc = new docusign.Document();
+    var base64Doc = Buffer.from(fileBytes).toString('base64');
+    doc.documentBase64 = base64Doc;
+    doc.name = 'TestFile.pdf';
+    doc.documentId = '1';
+
+    var docs = [];
+    docs.push(doc);
+    envDef.documents = docs;
+
+    // Add a recipient to sign the document
+    var signer = new docusign.Signer();
+    signer.email = userName;
+    var name = 'Pat Developer';
+    signer.name = name;
+    signer.recipientId = '1';
+
+    // this value represents the client's unique identifier for the signer
+    var clientUserId = '2939';
+    signer.clientUserId = clientUserId;
+
+    // create a signHere tab somewhere on the document for the signer to sign
+    // default unit of measurement is pixels, can be mms, cms, inches also
+    var signHere = new docusign.SignHere();
+    signHere.documentId = '1';
+    signHere.pageNumber = '1';
+    signHere.recipientId = '1';
+    signHere.xPosition = '100';
+    signHere.yPosition = '100';
+
+    // can have multiple tabs, so need to add to envelope as a single element list
+    var signHereTabs = [];
+    signHereTabs.push(signHere);
+    var tabs = new docusign.Tabs();
+    tabs.signHereTabs = signHereTabs;
+    signer.tabs = tabs;
+
+    // Above causes issue
+    envDef.recipients = new docusign.Recipients();
+    envDef.recipients.signers = [];
+    envDef.recipients.signers.push(signer);
+
+    // make the envelope with "created" (draft) status
+    envDef.status = 'created';
+
+    var envelopesApi = new docusign.EnvelopesApi(apiClient);
+
+    envelopesApi.createEnvelope(accountId, { envelopeDefinition: envDef }, function(error, envelopeSummary, _response) {
+      if (error) {
+        return done(error);
+      }
+
+      if (envelopeSummary) {
+        var returnUrl = 'http://www.docusign.com/developer-center';
+        var returnUrlRequest = new docusign.ReturnUrlRequest();
+        returnUrlRequest.returnUrl = returnUrl;
+
+        envelopesApi.createRecipientView(accountId, envelopeSummary.envelopeId, { recipientViewRequest: returnUrlRequest }, function(error, viewUrl, _response){
```

Comment:
You can increase the readability when you move the code from callback to a variable like

```
var callback = function(error, viewUrl, _response){ ... };
envelopesApi.createRecipientView(accountId, envelopeSummary.envelopeId, { recipientViewRequest: returnUrlRequest }, callback);

```